components/zcash_note_encryption/Cargo.toml

```diff
@@ -12,18 +12,22 @@ license = "MIT OR Apache-2.0"
 edition = "2018"
 
 [dependencies]
-blake2b_simd = "0.5"
-byteorder = "1"
-chacha20 = "0.8"
-chacha20poly1305 = "0.9"
-ff = "0.11"
-group = "0.11"
-rand_core = "0.6"
-subtle = "2.2.3"
+blake2b_simd = { version = "0.5", default-features = false }
```

Comment:
Hmm, this crate uses `std` to perform CPU detection for performance. We should keep that enabled by default, so I think we should add a `std` feature flag that isn't used by the crate itself, but just enables all the underlying dependency `std` flags.